corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -17,17 +17,22 @@
       </div>
     </div>
 
-    <% if (useGrid) { %>
+    <% if (useTiles) { %>
+      <% if (isMultiSelect && !hasNoItems) { %>
+      <input type="checkbox" id="select-all-tile-checkbox"/>
+      <label id="select-all-tile-checkbox-label" for="select-all-tile-checkbox">{% trans "Select All" %}</label>
+    <% } %>
     <section class="js-case-container clearfix list-cell-container-style"></section>
     <% } else { %>
     <div class="module-case-list-table-container">
       <table class="table module-table module-table-case-list">
-        <% if(!useTiles) { %>
+
         <thead>
         <tr>
           <% if (isMultiSelect && !hasNoItems) { %>
           <th class="module-case-list-header"><input type="checkbox" id="select-all-checkbox"/></th>
           <% } %>
+          <% if(!useTiles) { %>
```

Comment:
What is this check doing? It seems like it's already determined `useTiles` is false given this is in the `else` from line 26.

Reply:
Oh, or maybe that `else` is actually the alternative to `<% if (isMultiSelect && !hasNoItems) { %>` from line 21, and the whole thing is within line 20's `<% if (useTiles) { %>` ? I think there's maybe some indentation that could be cleared up in this section.

Reply:
No, I think you are right. This cannot be false at this point